r4/variables.bal

```diff
@@ -237,7 +236,172 @@ public final readonly & map<CommonSearchParameterDefinition> CONTROL_SEARCH_PARA
         default: (),
         effectiveLevel: SEARCH_PARAM_CATEGORY_SEARCH_RESULT,
         preProcessor: (),
-        postProcessor:  ()
+        postProcessor: ()
+    }
+};
+
+# Base operations for all resources.
+public final map<FHIROperationDefinition> & readonly BASE_RESOURCE_OPERATIONS = {
+    "convert": {
+        name: "convert",
+        instanceLevel: false,
```

Comment:
Where do we keep if the operation is active or not?

Reply:
An operation's active state is stored in the operation config. Here, inside the r4 package, we only keep the base operations' definitions (these are common operations for all resources). 

Configs for these base operations are constructed inside fhirr4's preprocessor at compile time (similar to common search parameters), 

https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/blob/1a90efd56b43845a23b6dbfa36bbccfbc1015456/fhirr4/ballerina/src/main/resources/fhirservice/fhir_preprocessor.bal#L65-L79

By default, all the base operations are active. This default config can be overridden by defining the base operation config in the service API config.